pkg/apis/core/validation/shoot.go

```diff
@@ -69,6 +69,7 @@ var (
 	)
 	availableWorkerCRINames = sets.NewString(
 		string(core.CRINameContainerD),
+		string(core.CRINameDocker),
```

Comment:
With such change, I think `docs/extensions/operatingsystemconfig.md` needs an update.

There are expressions like
> The only CRI supported at the moment is: "containerd"

> If the `.spec.cri` section exists then the `name` property is mandatory. The only valid value at the moment is `containerd`.

that now need an update.

Reply:
> With such change, I think docs/extensions/operatingsystemconfig.md needs an update.

Ah, this is where we hide this configuration, thanks for the pointer! I was looking at the API docs to find out more about supported values for `CRI.name`, but there wasn't anything mentioned. This is most likely out of the scope for this change, but do you feel `operatingsystemconfig.md` is the right place to mention the supported options for `CRI.name`?

Reply:
> I was looking at the API docs to find out more about supported values for CRI.name, but there wasn't anything mentioned.

We can mention the allowed values in the doc string of the corresponding field. For me, such change makes sense.

Reply:
> We can mention the allowed values in the doc string of the corresponding field. For me, such change makes sense.

Do you want to do it as part of this PR or do you want to leave it to another PR?

Reply:
I think a separate change would be better, given that #4222 also changes API docs and includes a `make generate` run.